src/test/java/org/embulk/parser/jsonpath/TestJsonpathParserPlugin.java

```diff
@@ -439,6 +440,54 @@ public void writeNilsWithJsonPath()
         }
     }
 
+    @Test
+    public void notArrayObject()
+            throws Exception
+    {
+        SchemaConfig schema = schema(column("__c0", STRING, config().set("path", "$._c0")));
+        ConfigSource config = this.config.deepCopy().set("columns", schema);
+
+        transaction(config, fileInput("{ \"_c0\": \"embulk\" }"));
+
+        List<Object[]> records = Pages.toObjects(schema.toSchema(), output.pages);
+        assertEquals(1, records.size());
+        assertEquals("embulk", records.get(0)[0]);
+    }
+
+    @Test
+    public void rootPathJsonIsObject()
+            throws Exception {
+        SchemaConfig schema = schema(column("__c0", STRING, config().set("path", "$._c0")));
+        ConfigSource config = this.config.deepCopy().set("columns", schema).set("root", "$[0].root");
+
+        transaction(config, fileInput(
+                "[",
+                "{ \"root\": { \"_c0\": \"embulk\" } }",
+                "]"
+        ));
+
+        List<Object[]> records = Pages.toObjects(schema.toSchema(), output.pages);
+        assertEquals(1, records.size());
+        assertEquals("embulk", records.get(0)[0]);
+    }
+
+    @Test
+    public void rootPathJsonIsNotArrayOrObject() {
```

Comment:
Could you fix `src/test/java/org/embulk/parser/jsonpath/TestJsonpathParserPlugin.java:475:50: '{' at column 50 should be on a new line. [LeftCurly]`?

```suggestion
    public void rootPathJsonIsNotArrayOrObject()
    {
```